schema/artist/index.js

```diff
@@ -129,6 +130,12 @@ const ArtistType = new GraphQLObjectType({
               auction_lots_count),
             for_sale_artworks: numeral(({ forsale_artworks_count }) =>
               forsale_artworks_count),
+            partner_shows: {
+              type: GraphQLInt,
```

Comment:
I'd prefer if all these count fields were of the numeral type (see above) (so you could format them in the query)